src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -202,25 +202,30 @@ SidebarLinks.defaultProps = defaultProps;
  * @param {Object} [report]
  * @returns {Object|undefined}
  */
-const reportSelector = report => report && ({
-    reportID: report.reportID,
-    participants: report.participants,
-    hasDraft: report.hasDraft,
-    isPinned: report.isPinned,
-    errorFields: {
-        addWorkspaceRoom: report.errorFields && report.errorFields.addWorkspaceRoom,
-    },
-    maxSequenceNumber: report.maxSequenceNumber,
-    lastReadSequenceNumber: report.lastReadSequenceNumber,
-    lastMessageText: report.lastMessageText,
-    lastMessageTimestamp: report.lastMessageTimestamp,
-    iouReportID: report.iouReportID,
-    hasOutstandingIOU: report.hasOutstandingIOU,
-    statusNum: report.statusNum,
-    stateNum: report.stateNum,
-    chatType: report.chatType,
-    policyID: report.policyID,
-});
+const chatReportSelector = (report) => {
+    if (ReportUtils.isIOUReport(report)) {
+        return null;
+    }
+    return report && ({
+        reportID: report.reportID,
+        participants: report.participants,
+        hasDraft: report.hasDraft,
+        isPinned: report.isPinned,
+        errorFields: {
+            addWorkspaceRoom: report.errorFields && report.errorFields.addWorkspaceRoom,
+        },
+        maxSequenceNumber: report.maxSequenceNumber,
+        lastReadSequenceNumber: report.lastReadSequenceNumber,
+        lastMessageText: report.lastMessageText,
+        lastMessageTimestamp: report.lastMessageTimestamp,
+        iouReportID: report.iouReportID,
+        hasOutstandingIOU: report.hasOutstandingIOU,
+        statusNum: report.statusNum,
+        stateNum: report.stateNum,
+        chatType: report.chatType,
+        policyID: report.policyID,
+    });
```

Comment:
NAB

```suggestion
    if (report! || ReportUtils.isIOUReport(report)) {
        return null;
    }
    return {
        reportID: report.reportID,
        participants: report.participants,
        hasDraft: report.hasDraft,
        isPinned: report.isPinned,
        errorFields: {
            addWorkspaceRoom: report.errorFields && report.errorFields.addWorkspaceRoom,
        },
        maxSequenceNumber: report.maxSequenceNumber,
        lastReadSequenceNumber: report.lastReadSequenceNumber,
        lastMessageText: report.lastMessageText,
        lastMessageTimestamp: report.lastMessageTimestamp,
        iouReportID: report.iouReportID,
        hasOutstandingIOU: report.hasOutstandingIOU,
        statusNum: report.statusNum,
        stateNum: report.stateNum,
        chatType: report.chatType,
        policyID: report.policyID,
    };
```

Reply:
I think this would return a slightly different report, but I'm not positive, tbh

Reply:
I think in the code as exist now, if report is falsey (i.e. `undefined`), it would return what ever is in there... so the function can potentially return `falsey values | null | {copied report object}`.. so yes, with this change your are changing a bit what the function is returning... but maybe that is desired? not sure either

Anyway.. up to you.. nab